tests/data/test2088

```diff
@@ -0,0 +1,52 @@
+<testcase>
+<info>
+<keywords>
+HTTPS
+HTTP GET
+Client Auth
+</keywords>
+</info>
+
+#
+# Server-side
+<reply>
+<data>
+HTTP/1.1 200 OK
+Date: Tue, 09 Nov 2010 14:49:00 GMT
+Server: test-server/fake
+Content-Length: 7
+
+MooMoo
+</data>
+</reply>
+
+#
+# Client-side
+<client>
+<features>
+SSL
+local-http
+</features>
+<server>
+https-mtls
+</server>
+<name>
+HTTPS GET with client authentication (mtls)
+</name>
+<command>
+--cacert %SRCDIR/certs/test-ca.crt --cert %SRCDIR/certs/test-client-cert.crt --key %SRCDIR/certs/test-client-cert.key https://localhost:%HTTPS-MTLSPORT/%TESTNUMBER
```

Comment:
Do you perhaps want `%CERTDIR` instead of `%SRCDIR` here? They are generated, right, so they are not in the source tree? /cc @vszakats

Reply:
Yes, `%CERTDIR` is the dir for generated certs.